snapcraft/plugins/make.py

```diff
@@ -42,6 +42,9 @@ def schema(cls):
         schema['properties']['makefile'] = {
             'type': 'string'
         }
+        schema['properties']['make_parameters'] = {
```

Comment:
Use a hyphen here instead of the underscore (it will become an underscore when filling out `self.options`)